From caec5c084f75f8060a34f360c16c3a0428cd93d6 Mon Sep 17 00:00:00 2001
From: Ratnesh Kumar Rai <ratnesh.kumar.rai@intel.com>
Date: Sat, 9 Apr 2022 09:57:38 +0530
Subject: [PATCH] Restore SG1 as a VPL platform

This restores SG1 as a VPL platform by reverting the change done as
part of commit [core] SG1 is not a VPL platform (#3478)

Change-Id: I32aed907bfa5658230ebb42846dc0651fba31aa4
Tracked-On: OAM-104545
Signed-off-by: Neeraj Solanki <neeraj.solanki@intel.com>
Signed-off-by: Ratnesh Kumar Rai <ratnesh.kumar.rai@intel.com>
Signed-off-by: yerriswamy <yerriswamy.kt@intel.com>
---
 _studio/mfx_lib/shared/include/mfx_platform_caps.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/_studio/mfx_lib/shared/include/mfx_platform_caps.h b/_studio/mfx_lib/shared/include/mfx_platform_caps.h
index b55ce386..84fb98fd 100644
--- a/_studio/mfx_lib/shared/include/mfx_platform_caps.h
+++ b/_studio/mfx_lib/shared/include/mfx_platform_caps.h
@@ -44,7 +44,7 @@ namespace CommonCaps {
 
     inline bool IsVplHW(eMFXHWType hw, mfxU32 deviceId)
     {
-        return hw >= MFX_HW_TGL_LP && deviceId != 0x4907;
+        return hw >= MFX_HW_TGL_LP;
     }
 
     inline bool CapsQueryOptimizationSupport(eMFXHWType platform)
-- 
2.17.1

